insights/parsers/scheduler.py

```diff
@@ -1,17 +1,50 @@
-from .. import parser, get_active_lines, Parser
+"""
+Scheduler - file ``/sys/block/*/queue/scheduler``
+=================================================================
```

Comment:
The length of `=` characters should be the same as the length of the module name line (for the documentation generation purpose).